### DISCUSSION TITLE
Allow to add parking meters in Organic Maps

### DISCUSSION THREAD

**Discussion by stefaniecg** [organicmaps/organicmaps#6088] [Ideas]
I like to contribute to the **OpenMaps** trough the **organic maps APP** by **adding** data to the map.

One thing i find very useful is to know if a **parking place is paid or not**. 
I would like to be able to add a **parking meter to the map**. 

Currently is not possible from the APP, only from the **OpenMaps web interface**. 
In the OpenMaps web app it is called: **parking ticket vending machine**

Please include this feature. 
Thank you so much, 

Stefanie
![image](https://cvws.icloud-content.com/B/AT1IaA-or9w_5HhMR5klb8jJgY4oAceLoulm4RnTb16zTKC0iCABFz5T/openmaps.png?o=AtmsuZtKuwHPwJVZwAJliFSmvxX7on04rf2yIH2vaiWv&v=1&x=3&a=CAogyPQPVgnhk4zZrXSC9rsmczxVCBPWrqV1JXLEGVpDhmQSaxCSw-bhqzEYkqDC46sxIgEAUgTJgY4oWgQBFz5TaiWVG5aQJk4PoTbZokZW6jC182ENcUy34U7_cT6g2kWN8ki9D9wcciX8dwZdjm37GkQM2owmrouMoRRIwJl4hsPWTbASUGFp8Q2kl26E&e=1695378608&fl=&r=b6cdaf32-f1fd-4400-8ef5-74f814178a34-1&k=-P_gNfqYokJq9Mvv-f_cLQ&ckc=com.apple.clouddocs&ckz=com.apple.CloudDocs&p=107&s=uxdMcQLF376mwQaZsRA3bRhUWcw&cd=i)

**Comment by map-per** [comment]
That's related to #4523 as parking ticket vending machines are mapped with two tags, `amenity=vending_machine` and `vending=parking_tickets`

**Reply by stefaniecg** [reply]
hi @map-per , so... not sure what this means? 
I cannot find neither `vending_machine` nor `parking_tickes` from the categories among one has available when adding data to the OpenStreets Map via the Organic Maps APP.

**Reply by map-per** [reply]
Yes, parking ticket vending machines are to complex for the OrganicMaps editor (see: #4523), so they can't be added with the app.
(`amenity=vending_machine` and `vending=parking_tickets` are the tags internally used in OpenStreetMaps (see https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dvending_machine))

**Reply by biodranik** [reply]
We need to fix the mentioned issue first, after that complex OSM types with more than one tag can be added.
